src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+import { getSelectedTab } from '../../utils/utils';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = getSelectedTab(tabs, selectedTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({ 'is-active': selectedTab.id
+              === tab.id })}
```

Comment:
Pay attention to your code style. Also, you compare `selectedTab.id !== tab.id` twice, you can create a variable in this case